backend/api/viewsets/ComplianceReport.py

```diff
@@ -406,12 +409,7 @@ def filter_draft(self, latest_supplemental):
 
     def get_latest_supplemental_reports(self):
         latest_supplementals = ComplianceReport.objects.filter(id__in=RawSQL("""
```

Comment:
Do we still need to use a raw query here or could we use just a Django query? is the raw query faster in this case?